Woof/Woof/View/ReusableViews/ButtonStyle/CapsuleWithWhiteText.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+/// A type that applies standard interaction behavior and a custom appearance to all buttons within a view hierarchy.
```

Comment:
typo
```suggestion
/// A type that applies standard interaction behaviour and a custom appearance to all buttons within a view hierarchy.
```

Reply:
There is no typo here. Both options are possible.